skglm/solvers/cd_solver.py

```diff
@@ -52,6 +56,9 @@ def cd_solver_path(X, y, datafit, penalty, alphas=None,
     return_n_iter : bool, optional
         If True, number of iterations along the path are returned.
 
+    solver : ('cd_ws'|'cd_gram'|'fista'), optional
```

Comment:
FISTA is not a CD solver, it's confusing to expose it to the user like this.
@mathurinm WDYT?